src/Model/ModelManager.php

```diff
@@ -182,7 +182,7 @@ public function findOneBy(string $class, array $criteria = []): ?object
     public function getEntityManager($class): EntityManagerInterface
     {
         if (\is_object($class)) {
-            $class = \get_class($class);
+            $class = ClassUtils::getClass($class);
```

Comment:
On lower deps, without this change, build fails. Do you know why @VincentLanglet ?

```
[critical] Uncaught PHP Exception Twig\Error\RuntimeError: "An exception has been thrown during the rendering of a template ("No entity manager defined for class Proxies\__CG__\Sonata\DoctrineORMAdminBundle\Tests\App\Entity\Author")." at /home/jsala/Projects/SonataDoctrineORMAdminBundle/vendor/sonata-project/admin-bundle/src/Resources/views/CRUD/Association/list_many_to_one.html.twig line 22
```

Reply:
No idea :/

Reply:
I'm pretty sure it may be related to changes in "doctrine/orm" around the proxies and lazy loading behaviors between the lower and the highest versions ([ref.](https://github.com/doctrine/orm/releases?q=proxy&expanded=true)).

Reply:
I think it may be related to some of these changes:
* doctrine/persistence#296
* doctrine/orm#10385

If they are fixing bugs, IMO we could try to update our constraints to disallow the affected versions.

Reply:
yeah would be good to check which versions exactly are causing this issue :confused: And then maybe bump the minimum requirement to get rid of the issue?

Reply:
It was 3.0.2 of doctrine/persistence, that avoids changing this line of code.

Reply:
nice :+1: